voter/src/main/java/org/terracotta/voter/ActiveVoter.java

```diff
@@ -143,6 +144,7 @@ public void registerAndHeartbeat(ExecutorService executorService, ClientVoterMan
                   long result = voterManager.vote(id, election);
                   LOGGER.info("own the vote, voting {} {} {}", voterManager.getTargetHostPort(), election, result);
                 } else if (owner.isConnected()) {
+                  owner = voteOwner.get();
```

Comment:
Unused? Or is there a logic here that I didn't understand?

Reply:
the owner of the vote may have changed if you have 3 threads running for 3+ servers.  This just resets the owner for the next round.